src/main/kotlin/org/expo/nothanks/utils/LobbyUtils.kt

```diff
@@ -165,12 +167,12 @@ fun Lobby.finishRound() {
     game = null
 }
 
-fun Lobby.createGame(): Game {
+fun Lobby.createGame(gameParams: GameParams): Game {
     val players = this.players.values.map {
         Player(
             id = it.id,
             number = it.number,
-            coins = params.defaultCoinsCount
+            coins = gameParams.coinsMap[this.players.size] ?: 11
```

Comment:
nit: `this` is already implied here
```suggestion
            coins = gameParams.coinsMap[players.size] ?: 11
```

Reply:
Здесь вариант Леши выглядит более безопасным
Хоть этот метод и должен быть вызван из потокобезопасного места, но внутри мы этого не видим, а players является копией this.players, не ссылкой на него. Поэтому лучше ссылаться на то поле, которому мы верим больше. Ну и ссылаться на коллекцию, по которой идет итерация лишний раз не стоит

Reply:
This is just syntactic sugar, there is no difference in multithreading, this extension is called on a [concrrete instance](https://kotlinlang.org/docs/extensions.html#extensions-are-resolved-statically).

Reply:
Тогда переменную players переименовать стоит , которая внутри объявляется, чтоб коллизий не было